datasets/msr_genomics_kbcomp/msr_genomics_kbcomp.py

```diff
@@ -0,0 +1,132 @@
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{toutanova-etal-2016-compositional,
+    title = "Compositional Learning of Embeddings for Relation Paths in Knowledge Base and Text",
+    author = "Toutanova, Kristina  and
+      Lin, Victoria  and
+      Yih, Wen-tau  and
+      Poon, Hoifung  and
+      Quirk, Chris",
+    booktitle = "Proceedings of the 54th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers)",
+    month = aug,
+    year = "2016",
+    address = "Berlin, Germany",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/P16-1136",
+    doi = "10.18653/v1/P16-1136",
+    pages = "1434--1444",
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+The database is derived from the NCI PID Pathway Interaction Database, and the textual mentions are extracted from cooccurring pairs of genes in PubMed abstracts, processed and annotated by Literome (Poon et al. 2014). This dataset was used in the paper “Compositional Learning of Embeddings for Relation Paths in Knowledge Bases and Text” (Toutanova, Lin, Yih, Poon, and Quirk, 2016).
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://msropendata.com/datasets/80b4f6e8-5d7c-4abc-9c79-2e51dfedd791"
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class MsrGenomicsKbcomp(datasets.GeneratorBasedBuilder):
+    """TODO(msr_genomics_kbcomp): Short description of my dataset."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+  To use msr_genomics_kbcomp you need to download it manually. Please go to its homepage (https://msropendata.com/datasets/80b4f6e8-5d7c-4abc-9c79-2e51dfedd791)and login. Extract all files in one folder and use the path folder in datasets.load_dataset('msr_genomics_kbcomp', data_dir='path/to/folder/folder_name')
+  """
+
+    def _info(self):
+        # TODO: This method pecifies the datasets.DatasetInfo object which contains informations and typings for the dataset
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # datasets.features.FeatureConnectors
+            features=datasets.Features(
+                {
+                    # These are the features of your dataset like images, labels ...
+                    "GENE1": datasets.Value("string"),
+                    "relation": datasets.Value("string"),
```

Comment:
What values can `relation` take ? If there are only a few of them we can use the ClassLabel feature type here.
You can find an example of usage in [conll2003](https://github.com/huggingface/datasets/blob/master/datasets/conll2003/conll2003.py)